md_doc/getting_started.md

```diff
@@ -2,63 +2,95 @@
 In this tutorial, you will learn how to deploy and use the RaSTA C library.
 This tutorial is written for Linux. If you are using another operating system, the commands should be similar.
 
-## 1. Getting the library
+## 1. Building the library
 
-Clone the repository.
+### Dependencies
 
+The following dependencies are required for building the RaSTA library and binaries:
 
-To compile the library you can use the Gradle task `build`
-```
-gradle build
-```
+* CUnit (package `libcunit1` or similar)
+* if `ENABLE_RASTA_OPAQUE` is enabled: libsodium (packages `libsodium-dev` and `pkgconf` or similar)
+* if `ENABLE_RASTA_TLS` is enabled: [WolfSSL](https://www.wolfssl.com/), built manually with the configuration flags `--enable-dtls --enable-debug --enable-certgen --enable-tls13 CFLAGS="-DHAVE_SECRET_CALLBACK" --enable-opensslextra`
```

Comment:
Actually, the custom options are only needed for enabling the secret callback and the code will compile even if this is disabled.

For debug purposes (dissecting recorded TLS traffic using Wireshark) it is useful to have this option enabled.

Maybe we can add this info here.